docs/get-started/deployment-and-usage.md

```diff
@@ -99,6 +99,51 @@ this approach does not guarantee running once on every node. For example,
 tainted, non-ready nodes or some other reasons in Job scheduling may cause some
 node(s) will run extra job instance(s) to satisfy the request.
 
+### Helm Chart
+
+This chart bootstraps a Node Feature deployment on a Kubernetes cluster using the Helm package manager.
+
+#### Prerequisites
+
+[Helm package manager](https://helm.sh/) should be installed.
+
+#### Installing the Chart
+
+To install the chart with the release name network-operator:
+
+```bash
+git clone https://github.com/kubernetes-sigs/node-feature-discovery/
+cd node-feature-discovery/deployment
+helm install network-operator ./network-operator/
+```
+
+The command deploys Node Feature Discovery on the Kubernetes cluster in the default configuration.
+The Configuration section describes how it can be configured during installation.
+
+#### Configuration
+
+You can override values from `values.yaml` and provide a file with custom values:
+
+```bash
+helm install network-operator ./network-operator/ -f values.yaml
+```
+
+To specify each parameter separately you can provide them to helm install command:
+
+```bash
+helm install network-operator ./network-operator/ --set nameOverride=NFDinstance --set master.replicaCount=2
+```
+
+#### Uninstalling the Chart
+
+To uninstall the `network-operator` deployment:
+
+```bash
+helm uninstall network-operator 
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
```

Comment:
Should we have a section which describes the chart parameters similarly to what we have in ?

https://github.com/Mellanox/network-operator/tree/master/deployment/network-operator#chart-parameters